en/docs/deploy-operate/observe/recipe-elk-stack.md

```
@@ -0,0 +1,430 @@
+---
+title: Recipe - ELK Stack (Complete Log Analytics)
+description: End-to-end setup of Elasticsearch, Logstash, Kibana for log aggregation and analysis.
+---
+
+# Recipe: ELK Stack (Complete Log Analytics)
+
+Complete log aggregation, full-text search, and visualization using the Elastic Stack. Perfect for organizations requiring advanced log analytics, compliance auditing, and long-term log retention.
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)